src/styles/scss/layout/layout.scss

```diff
@@ -0,0 +1,23 @@
+.layout {
+  max-width: 1440px;
+  margin: 0 auto;
+  .container {
+    max-width: 1200px;
+    margin: 0 auto;
+  }
+  @media (max-width: 1200px) {
+    .container {
+      max-width: 970px;
+    }
+  }
+  @media (max-width: 992px) {
+    .container {
+      max-width: 750px;
+    }
+  }
+  @media (max-width: 767px) {
+    .container {
+      max-width: none;
+    }
+  }
+}
```

Comment:
Please look at this code
```
.layout {
  max-width: 1440px;
  margin: 0 auto;
}

.container {
  max-width: none;
  margin: 0 auto;

  @media (min-width: 768px) {
    max-width: 750px;
  }

  @media (min-width: 992px) {
    max-width: 970px;
  }

  @media (min-width: 1200px) {
    max-width: 1200px;
  }
}
```

1. We don't have nesting
2. we don't have duplication
3. our styles should start from mobile and up. We using mobile first approach.

Reply:
in progress in header, footer, and landing story component.